templates/workspaces/base/terraform/storage.tf

```diff
@@ -18,6 +18,12 @@ resource "azurerm_storage_share" "shared_storage" {
   ]
 }
 
+resource "azurerm_storage_container" "shared_storage" {
```

Comment:
Can this container be created inside the ML flow workspace service, and named with an ML flow specific name? We should avoid adding things to the base workspace unless they are actually needed by multiple services and/or by all deployments.